sci-astronomy/stellarium/stellarium-25.3.recipe

```diff
@@ -62,9 +62,10 @@ BUILD_REQUIRES="
 	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
-	llvm20
+	llvm20$secondaryArchSuffix
 	cmd:cmake
-	cmd:clang++
+    cmd:clang >= 20
+	cmd:clang++ >= 20
```

Comment:
Checked this earlier to be sure, clang++ requires ld.lld which is already set to >= 20, so this ¨shouldn´t be needed, but doesn´t harm also